src/Components/Login/Login.tsx

```diff
@@ -15,12 +15,17 @@ export default function Login({ saveUserData }: any) {
     axios.post('http://upskilling-egypt.com:3003/api/v1/Users/Login', data)
 
       .then((response) => {
+console.log(response);
 
         localStorage.setItem("userToken", response?.data?.token)
+
         saveUserData()
-        navigate('/dashboard')
+  
+        
+       navigate('/dashboard')
 
       }).catch((error) => {
+console.log(error);
 
         toast(error?.response?.data?.message);
```

Comment:
Well, since i have no direct access to review the previous changes before what you made on this branch I'll I will write my comments here and refer to the line where the problem is!

In Line 63 (/^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/): This pattern you give the validate the email doesn't accept the Uppercase Letters and your email have an uppercase letter, so you will not be able to send the request to the server. a solution for that is the adjust the email pattern to make it accept uppercase letters, and you can consider the following as a solution (/^[a-zA-Z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,4}$/)